TWLight/users/templates/users/preferences.html

```diff
@@ -34,17 +34,43 @@ <h3>Preferences</h3>
       <div class="col-xs-12 col-sm-9">
         <form method="post" action="{% url 'password_change' %}">
           {% csrf_token %}
-          {{ password_form | crispy }}    
+          {{ password_form | crispy }}
           <button class="btn btn-default" type="submit">
             {% trans "Change password" %}
           </button>
-        </form>    
+        </form>
       </div>
     </div>
 
     <hr />
   {% endif %}
 
+  {% if user.userprofile.terms_of_use %}
+    <div class="row clearfix">
+      <div class="col-xs-12 col-sm-3">
+        {% comment %} Translators: This is the heading of the section of a user's
+        profile where they disagree their terms of use. {% endcomment %}
+        <strong>{% trans "Terms of use" %}</strong>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <form method="post" action="{% url 'terms' %}">
+          {% csrf_token %}
+          {{ terms_form | crispy }}
+
+          <button class="btn btn-default" type="submit">
+             {% comment %} Translators: This is a button for updating a user's term of use. {% endcomment %}
+             {% trans "Update" %}
+          </button>
+          </br>
+          </br>
+          {% comment %} Translators: This link is to Wikipedia Library Card Terms of Use and Privacy Statement. Wikipedia Library Card should not be translated. {% endcomment %}
+          <a href="{% url 'terms' %}"> {% trans "View Terms of Use" %}</a>
```

Comment:
Close. There's no 'Wikipedia Library Card' in the translatable string, you should remove that part.

Reply:
Thank you for catching that.  Please let me know if you prefer a different tone or verbiage.